inspire/base/templates/format/record/Inspire_HTML_detailed_macros.tpl

```diff
@@ -0,0 +1,327 @@
+      {% if collection_name == "HEP" %}
+        <span id="search-hep">Search Literature &gt;</span>
+      {% elif collection_name == "HEPNAMES" %}
+        <span id="search-hep">Search Authors &gt;</span>
+      {% elif collection_name == "CONFERENCES" %}
+        <span id="search-hep">Search Conferences &gt;</span>
+      {% elif collection_name == "JOB" %}
+        <span id="search-hep">Search Jobs &gt;</span>
+      {% endif %}
+    {% endfor %}
+  {% endif %}
+	<span id="title">{{ record_title() }}</span>
+{% endmacro %}
+
+{% macro record_title() %}
+	{% block title %}
+		{% if record['title'] | is_list %}
+        	{% for title in record['title'] %}
+          	{{ title['title']|capitalize }}
+          {% endfor %}
+        {% else %}
+        	{{ record['title']['title']|capitalize }}
+        {% endif %}
+	{% endblock %}
+{% endmacro %}
+
+{% macro record_buttons() %}
+	<a class="btn btn-default dropdown-toggle dropdown-cite cite-btn" type="button" id="dropdownMenu{{record['control_number']}}" data-recid="{{record['control_number']}}"  data-toggle="modal" data-target="#citeModal{{record['control_number']}}">
+		<i class="fa fa-quote-right"></i> Cite this record
+	</a>
+	{% if record.get('url') %}
+		{% set exists = [] %}
+		{% for url in record.get('url') %}
+			{% if url.get('url').endswith(".pdf") %}
+				{% if not exists %}
+					{% do exists.append(1) %}  
+						<a class="btn custom-btn btn-warning pdf-btn" href="{{ url.get('url') }}" target="_blank" role="button" title="View PDF"><i class="fa fa-eye"></i> View PDF</a>
+				{% endif %}
+			{% endif %}
+		{% endfor %}
+	{% endif %}
+{% endmacro %}
+
+{% macro record_publication_info() %}
+	{% if record.get('publication_info') | is_list %}
+			{% for pub_info in record.get('publication_info')%}
+				{{ show_publication_info(pub_info) }}
+			{% endfor %}
+	{% else %}
+		{% if record.get('publication_info') %}
+			{{ show_publication_info(pub_info) }}
+		{% endif %}
+	{% endif %}
+{% endmacro %}
+
+{% macro show_publication_info(pub_info) %}
+	{% if pub_info.get('journal_title') and pub_info.get('journal_volume') and pub_info.get('year') and pub_info.get('page_artid') %}
+		<span>{{ pub_info.get('journal_title') }} {{ pub_info.get('journal_volume') }} ({{pub_info.get('year')}}), {{ pub_info.get('page_artid') }}</span>
+	{% else %}
+		{% if pub_info.get('journal_title') or pub_info.get('journal_volume') or pub_info.get('year') or pub_info.get('page_artid') %}
+			<span>
+				{% if pub_info.get('journal_title') %}
+					{{ pub_info.get('journal_title') }} 
+				{% endif %}
+				{% if pub_info.get('journal_volume') %}
+					{{ pub_info.get('journal_volume') }} 
+				{% endif %}
+				{% if pub_info.get('year') %}
+					{{ pub_info.get('year') }} 
+				{% endif %}
+				{% if pub_info.get('page_artid') %}
+					{{ pub_info.get('page_artid') }} 
+				{% endif %}
+			</span>
+		{% endif %}
+	{% endif %}
+{% endmacro %}
+
+{% macro record_doi() %}
+	{% if record.get('doi') %}
```

Comment:
It is now `dois` https://github.com/inspirehep/inspire-next/blob/master/inspire/dojson/hep/schemas/hep-0.0.1.json#L211